packages/sn-filter-pane/src/ext/property-panel/data/data-panel/index.ts

```diff
@@ -63,26 +63,47 @@ export default function getDataPanelItems(env: IEnv) {
         },
       },
     },
-    searchMode: {
+    searchGroup: {
       type: 'items',
       translation: 'properties.filterpane.searchMode',
       grouped: false,
       items: {
+        showSearch: {
+          ref: 'searchEnabled',
+          type: 'boolean',
+          component: 'switch',
+          translation: 'properties.filterpane.searchField',
+          change: (data: EngineAPI.IGenericListProperties) => data?.qDef?.textAlign?.align || 'left',
```

Comment:
Should this `change` fn really be here? If so, what is the purpose with it?

Reply:
Hehe, my mistake. It should not be there!